Update its_my_personal_choice_to_eat_animals.md (#33)
veganhacktivists/animalsupportbot: knowledge/responses/its_my_personal_choice_to_eat_animals.md

So is eating animals a personal choice or is it a little more complex than that? 

> All acts come from personal choices. Some are inconsequential, like which socks to wear. Others have direct impacts on those who will suffer from that choice. It is your choice to eat animals. But by this logic it would then be morally justifiable to beat a dog or kick a cat, as it is a personal choice to do so. 

Reviewer: ```suggestion
All acts come from personal choices. Some are inconsequential, whereas others have direct impact on those who will suffer from that choice. It is your choice to eat animals. But by this logic it would then be morally justifiable to beat a dog or kick a cat, as it is a personal choice to do so. 
```

> If someone makes the personal choice to abuse a dog, does that make it morally justifiable? You may not realize, but you are speaking about non-human animals as they were not sentient beings, who will receive the full harm of your choice. The same way a dog would suffer if you decided you have fun kicking them. There is no difference between a dog, a cat, and a farmed animal when it comes to their capacity to feel pain, despair, terror and fear.

Reviewer: ```suggestion
If someone makes the personal choice to abuse a dog, does that make it morally justifiable? You may not realize it, but you are speaking about non-human animals as they were not sentient beings. As such, they will receive the full harm of your choice—in the same way a dog would suffer if you decided to kick them. There is no difference between a dog, a cat, and a farmed animal when it comes to their capacity to feel pain, despair, terror, and fear.
```

And remember, eating animals causes harm to humans too. Slaughterhouses and flesh processing plants bring about <a href="https://foodispower.org/human-labor-slavery/slaughterhouse-workers/">labor issues</a> that are otherwise absent from developed countries (poor compensation, safety issues, psychological trauma, exploitation of poor and marginalized communities).

> A personal choice will often affect others' personal choices too. When you declare it's yout personal choice to eat animal products, what about the personal choice of every other creature, human and non-human alike, whose life is treated as inferior and meaningless just  for minutes of sensorial pleasure that can easily be obtained from mure more ethical sources? 

Reviewer: ```suggestion
A personal choice will often affect others' personal choices too. When you declare it's your personal choice to eat animal products, you are not considering the personal choice of other creatures whose lives are treated as inferior and meaningless. Rather, their lives are reduced to minutes of sensory pleasure that can easily be obtained from more ethical plant-based sources.
```
